tmk_core/avr.mk

```diff
@@ -248,6 +248,14 @@ avrdude-split-right: $(BUILD_DIR)/$(TARGET).hex check-size cpfirmware
 usbasp: $(BUILD_DIR)/$(TARGET).hex check-size cpfirmware
 	avrdude -p $(MCU) -c usbasp -U flash:w:$(BUILD_DIR)/$(TARGET).hex
 
+BOOTLOADHID_PROGRAMMER ?= bootloadHID
+bootloadHID: $(BUILD_DIR)/$(TARGET).hex check-size cpfirmware
+	# bootloadHid executable has no cross platform detect methods
```

Comment:
Personally, I'd like to see this behind a define, like avrdude is, as that makes it easier to support in the `flash` target.

Reply:
Updated to define, if #6224 is merged first then I will come back and add the rest